### PR TITLE
supporting custom CA cert files

### PR DESCRIPTION
**Pull request by zeryx**
Added a path in the api client to enable custom ca cert files being used by the generated API client; this PR permits that functionality with the python client.

### CODE REVIEW

File: Test/client_test.py [hunk 1]
```diff
@@ -66,6 +66,20 @@ def test_get_build_logs_no_ssl(self):
             print(result)
         self.assertTrue("error" not in result)
 
+    def test_get_build_logs_customCert(self):
+        with open("./test.pem",'w'):
+            client = Algorithmia.client(api_key=os.environ.get('ALGORITHMIA_API_KEY'), ca_cert="./test.pem")
```
Comment (kennydaniel) [inline Test/client_test.py:71]: I think it should pickup api_key from environment variable automatically, no?

Also what exactly is this testing? The certificate is empty?
